src/ui/public/vislib/lib/axis/axis_labels.js

```diff
@@ -123,10 +123,16 @@ export function VislibAxisLabelsProvider() {
       return function (selection) {
         selection.each(function () {
           selection.selectAll('text')
-          .attr('style', function () {
-            const currentStyle = d3.select(this).attr('style');
-            return `${currentStyle} font-size: ${config.get('labels.fontSize')};`;
-          });
+            .attr('style', function () {
+              const currentStyle = d3.select(this).attr('style');
+              return `${currentStyle} font-size: ${config.get('labels.fontSize')};`;
+            })
+            .text(function (d) {
+              // fixes issue with missing labels for range term agg on Y axis
```

Comment:
no, d does not always contain the correct content ... it only does in the case of range. should probably investigate this further.